de.create3000.Titania.json

```diff
@@ -0,0 +1,427 @@
+{
+	"app-id": "de.create3000.titania",
+	"branch": "stable",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.32",
+	"sdk": "org.gnome.Sdk",
+	"command": "titania",
+	"finish-args": [
+		"--device=dri",
+		"--filesystem=home",
+		"--share=ipc",
+		"--share=network",
+		"--socket=pulseaudio",
+		"--socket=wayland",
+		"--socket=x11",
+		"--talk-name=de.create3000.*",
+		"--talk-name=de.create3000",
+		"--talk-name=org.freedesktop.Flatpak"
```

Comment:
What do you need this permission for?

Reply:
Thought --talk-name is the DBUS session name, and de.create3000. is my apps name. org.freedesktop.Flatpak: Titania has support for external tools. The user can create scripts or scriptlets which he then can execute. The scripts are then executed via flatpak-spawn, is this legal? I use GLX, didnt I need X11 acces?

Reply:
The access to my app is, because if you start Titania again, no new application window is openend, but only a new tab in the existing application is opened.

Reply:
The X11 stuff is fine, it's `org.freedesktop.Flatpak` that was the real issue. @barthalion @TingPing I'd appreciate your views as well if available.

Reply:
The talk name org.freedesktop.Flatpak is for the external tools that users can write:
![Bildschirmfoto von 2019-05-04 09-23-25](https://user-images.githubusercontent.com/2930196/57175649-b41fd400-6e4e-11e9-94b6-ca955f062ed9.png)
External Tools Editor

The tools are run with flatpak-spawn and provided some env like current directory and current file. I will be mostly some Perl or Python scripts to process the output or generate X3D. If not run on the host, they would be run within the flatpak, where no Python/Perl libraries are available. It is similar to VS Code tasks or GEdit External Tools.

Reply:
If it is not legal I would with heavy heart remove that.

Reply:
@nedrichards Removed --talk-name=org.freedesktop.Flatpak
https://github.com/create3000/flathub/blob/510c667e02aa10f6912f45c6269e19a32829794e/de.create3000.titania.json#L8-L18